CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `Status` type was added to the `go.opentelemetry.io/otel/sdk/trace` package to represent the status of a span. (#1874)
 - The `SpanStub` type and its associated functions were added to the `go.opentelemetry.io/otel/sdk/trace/tracetest` package.
   This type can be used as a testing replacement for the `SpanSnapshot` that was removed from the `go.opentelemetry.io/otel/sdk/trace` package. (#1873)
+- Semantic convention generation tooling has been added. (#1891)
```

Comment:
Is this something that OTel API/SDK users should be aware of? If so then probably some docs for `semconv-gen` should be added.

Reply:
No, not really.  It's an internal too meant for maintainer use.  This line can probably be removed.

Reply:
Let's remove it then  🚀